public/content/documentation/native/patterns/tidbit.md

```diff
@@ -0,0 +1,271 @@
+  Your browser does not support the video tag.
+</video>
+
+### Android Talkback
+<video controls>
+  <source src="media/video/native/tidbit/tidbit_AndroidTalkback.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+## iOS Developer Notes
+
+   - Initial screen reader focus can be on the entire tidbit and announce everything with or without a Call To Action (CTA). Or, the text and CTA can have separate focus.
+   - Meaningful images should have an accessibility label
+  
+### Name
+
+   - Programmatic name describes the purpose of the control (if CTA is available)
+   - If visible text label exists, the programmatic name should match the visible text label.
+        - **Note:** Setting a programmatic name while a visible text label exists may cause VoiceOver to duplicate the announcement of the name. If this happens, hide the visible text label from VoiceOver recognition.
+
+#### UIKit
+
+   - You can programmatically set the visible label with `setTitle()`.
+        - The _component title will overwrite the _component `accessibilityLabel`.
+   - If a visible label is not applicable in this case, set the _component `accessibilityLabel` to the label of your choice.
+        - To do this in Interface Builder, set the label using the Identity Inspector
+   - To hide labels from VoiceOver programmatically, set the label's `isAccessibilityElement` property to `false`
+   - To hide labels from VoiceOver using Interface Builder, uncheck `Accessibility Enabled` in the Identity Inspector.
+
+#### SwiftUI
+
+   - If no visible label, use view modifier `accessibilityLabel(_:)`
+
+### Role
+
+   - Please use guidance for the Button control if a CTA is applicable
+
+### Groupings
+
+   - Group visible label with button, if applicable, to provide a programmatic name for the button.
+
+#### UIKit
+
+   - Ensure that the child elements of the overarching view you want to group in has their `isAccessibilityElement` properties set to false.
+   - Set `isAccessibilityElement` to `true` for the parent view. Then, adjust `accessibilityLabel` and `accessibilityTraits` accordingly.
+        - If frame does not exist due to custom button, use `accessibilityFrameInContainer` to set the custom control’s frame to the parent view’s container or view of your choice.
+            - You can also unionize two frames with `frame.union` (i.e. `titleLabel.frame.union(subtitleLabel.frame)`).
+        - Use `shouldGroupAccessibilityElement` for a precise order if the native order should be disrupted.
+        - Use `shouldGroupAccessibilityChildren` to indicate whether VoiceOver must group its children views. This allows making unique vocalizations or define a particular reading order for a part of the page.
+
+#### SwiftUI
+
+   - Use view modifier `accessibilityElement(children: .combine)` to merge the child accessibility element’s properties into the new `accessibilityElement`
+
+### State 
+
+#### UIKit
```

Comment:
Same comment here as in the Android Dev Notes: 

"Swift UI", "UIKit", and etc. have been bullets under "Name", "Role", etc. in other components. 

Not a huge deal so it's up to you but just noting for consistency.